src/main/java/seedu/jarvis/model/consultation/Consultation.java

```diff
@@ -28,6 +30,10 @@ public LocalDateTime getDateAndTime() {
         return dateAndTime;
     }
 
+    public String getFormattedDateTime() {
+        return dateAndTime.format(DateTimeFormatter.ofLocalizedDateTime(FormatStyle.SHORT));
+    }
+
```

Comment:
Good method provided to abstract the details!

Reply:
thanks for making the date and time consistent

Reply:
Agreed, thanks for the effort!